### DISCUSSION TITLE
I am seeing a disturbing trend in GitHub of placing advertisements disguised as repositories <title>

### DISCUSSION THREAD

**Discussion by dsbrown** [community/community#45471] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am seeing a disturbing trend in GitHub of placing advertisements disguised as repositories.

Take for example this repo: https://github.com/Zyadon7/dupeguru

There is no code, not even an API specification, its just an advertisement for his product.  In my opinion, this is not productive for GitHub or the community and if it continues GitHub will just become a crappy and less effective version of Softpedia. My opinion is, that GitHub must have certain standards, and blatant advertising such as this is far below that standard and will lead to the eventual demise of this product and community.

My suggestion to rectify this is:

1) Make it clear that this is not allowed. If you want exposure for your product, you must include the code for that product.
2) Put a "report this repository" button on every repo page to make reporting easy.
3) Implement a three strike policy to reduce abuse. 

What say you?

**Comment by TolinSimpson** [comment]
I haven't seen this to be a widespread issue and that example repo didn't even include any tags for exposure. 

Personally, I'd like to keep the ability to advertise my other repos or products from a repository's page. i.e. similar products. I will always actually include code if I do and wouldn't want "the rules" preventing me from utilizing an advertising method because of a minority of repos with the sole purpose of advertising. 

It should also be considered that report systems are subject to abuse, I would hate for competition to send bots to come in and get my repo shut down. Just look at YouTube and all the copyright strike nonsense plaguing creators there.

I say no.

**Comment by spenserblack** [comment]
>There is no code, not even an API specification, its just an advertisement for his product.

>If you want exposure for your product, you must include the code for that product.

There are probably plenty of repos out there that are *future ideas,* and have little to no code. I've made a few myself, where I just write a README describing the general idea of what I want to do. These rules seem like they could unfairly punish "to do" projects.

Also, [GitHub's docs are public](https://github.com/github/docs), so one could say that GitHub is hosting its own advertisement ("here's all the cool things you can do with GitHub!"). Yet [GitHub itself is closed-source](https://stackoverflow.com/a/24256558). Doesn't that mean GitHub itself would be breaking these proposed rules? :thinking:
